src/app/pages/about-us/about-us.module.ts

```diff
@@ -1,10 +1,11 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { AboutUsComponent } from './components';
-import { SharedModule } from 'src/shared/shared.module';
-import { TranslateModule } from '@ngx-translate/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ReactiveFormsModule } from '@angular/forms';
+import { TranslateModule } from '@ngx-translate/core';
+import { AboutUsComponent, FeedbackFormComponent, RatingComponent} from './components';
 import { SvgIconsModule } from '../../../core';
+import { SharedModule } from 'src/shared/shared.module';
```

Comment:
1. create alias for "core module" in `tsconfig.json` file. for reference, check the shared module in this file and read this [article](https://medium.com/@thepawanluhana/angular-alias-import-with-typescript-d8e79dd9e5d)
2. upon importing "shared module" use the already existing alias. check other files that are importing "shared module" for reference